### PR TITLE
Update idle-session-timeout-web-apps.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the document as the Azure AD conditional access for different users and groups are not available for Office.com or any other admin centers. From my testing it is restricted to SP and EXO.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8132

File: microsoft-365/admin/manage/idle-session-timeout-web-apps.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ When a user has been inactive in Microsoft 365 web apps for the time period you
 
 - Idle session timeout works on a per-browser session basis. A user’s activity on Microsoft Edge is treated differently than their activity in other browsers such as Google Chrome. Users will be signed out from all tabs corresponding to their account within that browser session.
 
-- Once you turn on idle session timeout, it applies to your entire organization and can't be scoped to specific users, organizational units, or groups. Use [Azure AD Conditional Access](/azure/active-directory/conditional-access/) for policies for different users and groups.
+- Once you turn on idle session timeout, it applies to your entire organization and can't be scoped to specific users, organizational units, or groups. Use [Azure AD Conditional Access](/azure/active-directory/conditional-access/)  policies for different users and groups to access SharePoint and Exchange Online.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/idle-session-timeout-web-apps.md]:
```suggestion
- Once you turn on idle session timeout, it applies to your entire organization and can't be scoped to specific users, organizational units, or groups. Use [Azure AD Conditional Access](/azure/active-directory/conditional-access/) policies for different users and groups to access SharePoint and Exchange Online.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Pull request merged**